tests/rbd_mirror/test_rbd_mirror_snapshot.py

```diff
@@ -41,10 +41,15 @@ def run(**kw):
         snapshot_schedule_level = config.get("snapshot_schedule_level")
         if not snapshot_schedule_level or snapshot_schedule_level == "cluster":
             mirror1.mirror_snapshot_schedule_add()
+            mirror1.mirror_snapshot_schedule_add(interval="24h")
```

Comment:
Can we have a shorter interval say 5mins or so, so that if we verify snapshots created after the interval (which in my opinion we should as part of verify snapshot schedule, we will be able to see a snapshot without having to wait for 24h since such a long duration is not acceptable in cephci)

Reply:
Also have interval given as an input as well, and add a default value only when input is not given?

Reply:
yes default value is thereby **1m** if it's not given 
https://github.com/red-hat-storage/cephci/blob/20d9f3b0c4a403d36959e19b692d07befb0d27a9/tests/rbd_mirror/rbd_mirror_utils.py#L326
minutes value is already added in line 43 hence added hours value as 24h as multiple schedulers,
Here basically we are checking command execution success or failure also some on removal of snapshot schedule and verification also added to this interval.

Reply:
Yes changed to `1h` as interval instead of `24h`